diff.py

```diff
@@ -697,12 +700,19 @@ def apply_format(self, chunk: str, f: Format) -> str:
             ]
         else:
             static_assert_unreachable(f)
-        return f"{ansi_code}{chunk}{Style.RESET_ALL}"
+        return f"{ansi_code}{chunk}{Fore.RESET}"
```

Comment:
I think the `Style.BRIGHT` & `STYLE_UNDERLINE` also need to be disabled here? Maybe add `Style.NORMAL` and `STYLE_NO_UNDERLINE = "\u001b[24m"`?